2018-10-31T10:02:44.678Z - [32minfo[39m: Data Base connection success ...
2018-10-31T10:02:44.734Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 401 but got 200 date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=171737088, heapTotal=122327040, heapUsed=99890904, external=1163832, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=75, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 401 but got 200,     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:75:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.753Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 403 but got 200 date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=175792128, heapTotal=126521344, heapUsed=92965616, external=75358, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=87, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 403 but got 200,     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:87:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.762Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176062464, heapTotal=126521344, heapUsed=93663872, external=76567, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.end, line=119, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.get.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:119:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.766Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176062464, heapTotal=126521344, heapUsed=94154896, external=85917, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.get.set.end, line=145, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameters', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.get.set.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:145:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.780Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 204 but got 200 date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176062464, heapTotal=126521344, heapUsed=96762544, external=86887, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.end, line=173, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 204 but got 200,     at chai.request.put.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:173:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.789Z - [31merror[39m: uncaughtException: expected { Object (domain, _events, ...) } to have status code 404 but got 200 date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176062464, heapTotal=126521344, heapUsed=99168336, external=95585, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=33, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.put.send.end, line=188, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (domain, _events, ...) } to have status code 404 but got 200,     at chai.request.put.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:188:33),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.798Z - [31merror[39m: uncaughtException: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176332800, heapTotal=126521344, heapUsed=99708608, external=96593, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=224, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false, platform: 'magento' } to have property 'status' of true, but got false,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:224:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.801Z - [31merror[39m: uncaughtException: expected { status: false } to be empty date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176332800, heapTotal=127569920, heapUsed=100047752, external=105267, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=66, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=234, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { status: false } to be empty,     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:234:66),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
2018-10-31T10:02:44.806Z - [31merror[39m: uncaughtException: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"' date=Wed Oct 31 2018 14:02:44 GMT+0400 (+04), pid=22381, uid=10891, gid=10030, cwd=/home/armine_ha/Desktop/integration-microservice, execPath=/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, version=v6.11.3, argv=[/home/armine_ha/.nvm/versions/node/v6.11.3/bin/node, /home/armine_ha/Desktop/integration-microservice/node_modules/mocha/bin/_mocha, --reporter, spec, --compilers, ts:ts-node/register, tests/**/*.test.ts], rss=176332800, heapTotal=127569920, heapUsed=100454752, external=106015, loadavg=[1.0048828125, 0.79443359375, 0.9072265625], uptime=101265, trace=[column=44, file=/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts, function=chai.request.del.send.end, line=250, method=end, native=false, column=12, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=Test.Request.callback, line=716, method=callback, native=false, column=18, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js, function=parser, line=916, method=null, native=false, column=7, file=/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js, function=IncomingMessage.res.on.e, line=19, method=e, native=false, column=20, file=events.js, function=emitNone, line=91, method=null, native=false, column=7, file=events.js, function=IncomingMessage.emit, line=185, method=emit, native=false, column=12, file=_stream_readable.js, function=endReadableNT, line=974, method=null, native=false, column=11, file=internal/process/next_tick.js, function=_combinedTickCallback, line=80, method=null, native=false, column=9, file=internal/process/next_tick.js, function=process._tickCallback, line=104, method=_tickCallback, native=false], stack=[AssertionError: expected { Object (code, message) } to have property 'message' of 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"', but got 'Invalid request parameterserror: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"error: invalid input value for enum platforms: "Zaza"',     at chai.request.del.send.end (/home/armine_ha/Desktop/integration-microservice/tests/integrationRouter.test.ts:250:44),     at Test.Request.callback (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:716:12),     at parser (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/index.js:916:18),     at IncomingMessage.res.on.e (/home/armine_ha/Desktop/integration-microservice/node_modules/superagent/lib/node/parsers/json.js:19:7),     at emitNone (events.js:91:20),     at IncomingMessage.emit (events.js:185:7),     at endReadableNT (_stream_readable.js:974:12),     at _combinedTickCallback (internal/process/next_tick.js:80:11),     at process._tickCallback (internal/process/next_tick.js:104:9)]
